src/main/java/BlackjackController.java

```diff
@@ -3,230 +3,207 @@
 import javafx.scene.control.Label;
 import javafx.scene.control.TextField;
 
+import java.awt.*;
+import java.io.IOException;
 import java.util.List;
 
 public class BlackjackController {
 
     @FXML
-    List<Label> myCardsLabels, dealerCardsLabels;
+    List<Label> playerCards, dealerCards;
     @FXML
-    TextField betAmount;
+    TextField betTextField;
     @FXML
-    Label totalBetLabel, myCardsTotalLabel, dealerCardsTotalLabel,
-            messageLabel, earningsLabel;
-    int totalBet = 0;
-    int earnings = 0;
-    int myCardsTotal = 0;
-    int dealerCardsTotal = 0;
-    int myOnes = 0;
-    int dealerOnes = 0;
-    int dealerFaceDown = 0;
-    boolean gameOver = false;
-    boolean dealerBlackjack = false;
-    boolean meBlackjack = false;
-
-    private final Deck deck;
-
-    public BlackjackController(Deck deck) {
-        this.deck = deck;
+    Label playerCardsTotalLabel, dealerCardsTotalLabel,
+            messageLabel, totalEarningsLabel;
+    double betAmount;
+    double earnings, totalEarnings;
+    int playerTotal;
+    int dealerTotal;
+    int playerAces;
+    int dealerAces;
+    int dealerFaceDown;
+    boolean gameInProgress;
+
+    private final Deck deck = new Deck();
+
+    @FXML
+    public void initialize(){
+        messageLabel.setText("Betting is optional");
+        gameInProgress = false;
+        betAmount = 0;
+        earnings = 0;
+        totalEarnings = 0;
+        playerTotal = 0;
+        dealerTotal = 0;
+        playerAces = 0;
+        dealerAces = 0;
+        dealerFaceDown = 0;
     }
 
     public void onStartRound(ActionEvent actionEvent) {
-        if (!gameOver) {
+        if (gameInProgress == true) {
             messageLabel.setText("There is a game in progress");
         } else {
-            //player must bet to join round
-            while (betAmount.getText() == null) { //validate input.   .equals("") ??
-                messageLabel.setText("You must place a bet to enter the round.");
+            clearTable();
+            if (betTextField.getText() == null) {
```

Comment:
We did not discuss betting. It's a decent feature, however if you're not starting out with any money it really doesn't make sense.